src/User.ts

```diff
@@ -13,13 +13,15 @@ import { Amount, RawTxObject, Signature, UserObject } from './typings'
  * related methods.
  */
 export class User {
-  private signer: TxSigner
+  private signer: TLWallet
   private transaction: Transaction
   private provider: TLProvider
 
+  private defaultPassword = 'ts'
```

Comment:
I am following the implementation we did with eth-lightwallet here. Do we want to make the password mandatory or optional? It is mandatory in ethers.js

Reply:
https://github.com/trustlines-network/clientlib/issues/189